src/modules/margin_tokens/MarginToken.ts

```diff
@@ -93,6 +93,23 @@ export default abstract class MarginToken {
     exchangeWrapper: ExchangeWrapper,
     orderData: string,
     options: ContractCallOptions,
+  ): Promise<object> {
+    return this.contracts.callContractFunction(
+      this.contracts.erc20PositionWithdrawerV2.withdraw,
+      { ...options, from: withdrawer },
+      marginTokenAddress,
+      this.contracts.weth9.address,
+      exchangeWrapper.getAddress(),
+      orderData,
+    );
+  }
```

Comment:
I think we need one more function (only for `erc20PositionWithdrawerV2`) that wraps the following function signature:

```
function withdrawAsEth(
    address erc20Position
){...}
```

Reply:
All this function does is withdraw WETH and convert it to ETH and then send it back to the person. These previous ones take DAI and convert it to ETH

Reply:
Maybe think about renaming the other ones (`withdrawETHPayoutV1`) since they could also be used to withdraw to non-eth? Not sure @AntonioJuliano  thoughts?

Reply:
Yes we need to support `withdrawAsEth` too. I don't fully understand your last comment brendan